### DISCUSSION TITLE
Fonts are regularly stripped from my style.css file

### DISCUSSION THREAD

**Discussion by 16am** [gregsullivan/_tw#66] [Q&A] [answered]
Hi everybody, I've just started playing around with the theme builder and I had no particular issues to start developing and having fun with it.

But I just wonder why I fail about adding fonts.

What I've done is to place my imports
`@import url('https://fonts.googleapis.com/css2?family=Roboto+Serif:ital,opsz@0,8..144;1,8..144&display=swap');`
at the beginning of the style.css file, but after a couple of times the watcher runs, all the imports are gone and the font-family properties are gone too.

I really don't get why, can anybody tell me where I'm wrong?

Thank you!

**Comment by gregsullivan** [comment, marked as answer]
Hello—Thanks for posting this!

Because the `style.css` file is generated, you can't make edits to it directly without them being overwritten the next time Tailwind writes to the file.

Instead, you can make changes in the `./tailwind` folder. I think the `tailwind.css` file is likely your best bet in this instance.

Hope that helps!

**Reply by 16am** [reply]
That makes a lot of sense, thank you!

**Reply by gregsullivan** [reply]
You're very welcome!
